profiles/package.mask

```diff
@@ -33,6 +33,29 @@
 
 #--- END OF EXAMPLES ---
 
+# David Seifert <soap@gentoo.org> (2022-01-09)
+# No other distro packages these, long dead and unmaintained.
```

Comment:
You may want to point out that at least some of those are now part of sys-cluster/rdma-core (sys-fabric/infiniband-diags for example).

Reply:
thanks, I will improve the message once the PR is green